README.md

```diff
@@ -110,6 +110,59 @@ end
 
 ```
 
+#### Testing that a ruby method is called with the proper arguments
```

Comment:
Does this mean a puppet function written in ruby? If so I'd rather have that in the section title and a longer description before the example.

Reply:
This can be used for any ruby code under test - custom function, custom type/provider, custom fact (if it's possible to test a custom fact), etc.

If you want a better description, I'll have to try to come back to this in a few days when I'm feeling better.